doctr/models/recognition/sar.py

```diff
@@ -127,6 +127,7 @@ def call(
         _, states = self.lstm_decoder(holistic, states, **kwargs)
         sos_symbol = self.vocab_size + 1
         symbol = sos_symbol * tf.ones(shape=(batch_size,), dtype=tf.int32)
```

Comment:
Sorry, last request: could you change this to:
```
symbol = tf.fill(batch_size, sos_symbol)
```
to avoid the unnecessary multiplication